db/schema.rb

```diff
@@ -40,15 +40,12 @@
     t.boolean  "system",               limit: 1,     default: false, null: false
   end
 
-  create_table "assessment_categories", force: :cascade do |t|
-    t.string  "name",      limit: 255
-    t.integer "course_id", limit: 4
-  end
-
   create_table "assessment_user_data", force: :cascade do |t|
     t.integer  "course_user_datum_id", limit: 4,               null: false
     t.integer  "assessment_id",        limit: 4,               null: false
     t.integer  "latest_submission_id", limit: 4
+    t.integer  "cgdub",                limit: 4
+    t.integer  "special_type",         limit: 4,   default: 0, null: false
```

Comment:
Yeah, but I think we should do it in another branch. I'll try to cleanup for now.